### ISSUE TITLE
Add task color

### ISSUE THREAD

**Issue by oliviermaridat** [kanboard/plugin-rocketchat#4]
Is it possible to use "attachments" instead of "text" to add the related color of the task to the message? I do not know how to get properly the "task color" value, otherwise, I could do the modification.

See: https://api.slack.com/docs/message-attachments

**Comment by fguillot** [comment]
I'm not sure if attachments are compatible with RocketChat?

**Comment by oliviermaridat** [comment]
Yes they are, they are given as an example into the RocketChat configuration (with the same syntax as Slack), but there is no way to find a doc about it in RocketChat website...

**Comment by monspid** [comment]
Maybe you should try make it more sexy:
Where the color on the left is the color of the task - in this situation it is blue.
![image](https://user-images.githubusercontent.com/18348137/31807252-5d3d4636-b56d-11e7-96e1-30457a17f4da.png)
It is how to get that:
![image](https://user-images.githubusercontent.com/18348137/31807287-976f353a-b56d-11e7-88e5-3e908b9eccd5.png)
```
text: *[Project_Name]* [Name of a task with a link](https://my.kanboard.com/project/1/task/51)
attachments : [
    author_name: "Peter Jenkins", 
    author_icon: "PJ_icon.jpg", 
    text: "assigned: Thomas Wat\nstatus: *moved* the task *#51* to the column *Done*"
    image_url: "https://sth.com/images/kanboard.png"
    color: "#0000ff" - blue in hex
]
```
